Lab_1.md

```diff
@@ -43,9 +43,76 @@ The last step
 
 #### 1. Fizzbuzz
 * starting at one and incrementing by one to a number that is given, print out either the number, or if the number is divisible by 2, print "Fizz", and if the number is divisible by 3 print "Buzz" 
-
+~~~
+----------------------
+import user number
+---------------------
+         |
+         v
+        / \
+      /     \
+    /         \
+  /             \
+ divide the number by 2
+/ is there a remander ? __>____ no_>__Print "Fizz"____>_______
```

Comment:
What about 6?  Just asking to bring up that there is often more to a problem/question than we first see.  These look good.